libs/core/src/lib/form/form-item/form-item.component.scss

```diff
@@ -4,3 +4,5 @@
 @import "~fundamental-styles/dist/radio";
 @import "~fundamental-styles/dist/textarea";
 @import "~fundamental-styles/dist/form-select";
+@import "~fundamental-styles/dist/fieldset";
+
```

Comment:
@InnaAtanasova Not sure if this should be importing all of these styles?

Reply:
I removed unused.